src/limitations/prerelease_versions.md

```diff
@@ -90,23 +76,23 @@ matched to:
 
 ```rust
 DoubleRange {
-    normal_range: Range::none,
-    prerelease_range: Range::between("2.0.0-alpha", "2.0.0"),
+    normal_range: Ranges::empty(),
+    prerelease_range: Ranges::between("2.0.0-alpha", "2.0.0"),
 }
 ```
 
 And the constraint `2.0.0-alpha <= v < 2.1.0` would have the same
 `prerelease_range` but would have `2.0.0 <= v < 2.1.0` for the normal range.
-Those constraints could also be intrepreted differently since not all
+Those constraints could also be interpreted differently since not all
 pre-release systems work the same. But the important property is that this
-enable a separation of the dimensions that do not behave consistently with
+enables a separation of the dimensions that do not behave consistently with
 regard to the mathematical properties of the sets manipulated.
 
-All this is under ongoing experimentations, to try reaching a sweet spot
-API-wise and performance-wise. If you are eager to experiment with all the
-extensions and limitations mentionned in this section of the guide for your
-dependency provider, don't hesitate to reach out to us in our [zulip
-stream][zulip] or in [GitHub issues][issues] to let us know how it went!
+All this needs more experimentation, to try reaching a sweet spot API-wise and
```

Comment:
Do we want to mention that this is successfully being used for modeling rust dependencies implemented in https://github.com/pubgrub-rs/semver-pubgrub